README.md

```diff
@@ -6,5 +6,5 @@ Chocolatey PyPy
 This repository contains configuration for the PyPy packages on
 Chocolatey:
 
-- Python 2.7.13 [![Chocolatey pypy2](https://img.shields.io/chocolatey/v/python.pypy.svg)](https://chocolatey.org/packages/python.pypy)
-- Python 3.6.1 [![Chocolatey pypy3](https://img.shields.io/chocolatey/v/pypy3.svg)](https://chocolatey.org/packages/pypy3)
+- Python 2.7 [![Chocolatey pypy2](https://img.shields.io/chocolatey/v/python.pypy.svg)](https://chocolatey.org/packages/python.pypy)
```

Comment:
Why did you drop the patch versions here?

Reply:
I just installed 7.3.1 and the versions are 2.7.13 and 3.6.9
```
$ pyenv local pypy2.7-7.3.1
$ python
Python 2.7.13 (d0d41085a83c, Apr 06 2020, 17:09:41)
[PyPy 7.3.1 with GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>> 
$ pyenv local pypy3.6-7.3.1
$ python
Python 3.6.9 (2ad108f17bdb, Apr 07 2020, 02:59:26)
[PyPy 7.3.1 with GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

Reply:
I didn't see it anywhere on the pypy website so I thought it's best to remove this obscure patch version info :)

Reply:
It's not exactly obscure if the interpreter advertises the patch version?

Reply:
Alright fine, fixed now.